README.md

```diff
@@ -128,15 +165,36 @@ OakMap<K,V> oak = builder.build();
 You are welcome to take a look on the OakMap's [full API](https://github.com/yahoo/Oak/wiki/Full-API).
```

Comment:
It almost a must to update *full API* section as well... This will exactly show the API...

Reply:
Important not to forget when committing. Maybe to add Performance Evaluation at the same time.